app/build.gradle

```
@@ -75,20 +75,29 @@ connectedCheck {
 }
 dependencies {
 
+    testImplementation 'org.junit.jupiter:junit-jupiter'
```

```suggestion
    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.1'
```

Should fix the CI (and also my local build teeheehee)

---

this no longer relevant